mathml/presentation-markup/tokens/tokens-rendering-from-in-flow.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>tokens rendering from in-flow children</title>
+    <link rel="help" href="https://w3c.github.io/mathml-core/#token-elements">
+    <meta name="assert" content="Verify rendering of tokens is only affected by in-flow children.">
+    <style>
+      .oof1 {
+          left: 0;
+          position: absolute;
+      }
+      .oof2 {
+          left: 0;
+          position: fixed;
+      }
+      .box {
+          display: inline-block;
+          width: 10px;
+          height: 10px;
+      }
+    </style>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script>
+      setup({ explicit_done: true });
+      window.addEventListener("load", runTests);
+
+      function runTests() {
+        let container = document.querySelector("#container");
+        const epsilon = 1;
+
+        for (let math of container.children) {
+          let tagName = el.id;
+          let element = math.firstElementChild;
+          let reference = element.nextElementSibling;
+
+          test(function() {
+            compareLayout(element, reference, epsilon);
+          }, `Rendering of ${tagName} should only be affected by in-flow children`);
+        }
+
+        done();
+      }
+    </script>
+  </head>
+  <body>
+    <div id="log"></div>
+    <div id="container">
+      <math id="mo">
+        <mo><span class="box" style="background: lightblue"></span><span class="oof1 box" style="top: 0"><span class="oof2 box" style="top: 0"></span><span class="box" style="background: pink"></span><span class="box" style="background: yellow"></span><span class="oof1 box" style="top: 10px;"></span><span class="oof2 box" style="top: 10px;"></span></mo>
```

Comment:
and what we need style="top: ..." here and below

Reply:
OK, fixed